main.js

```diff
@@ -0,0 +1,12 @@
+
+    var question1 = prompt("日本の首都は？");
+    
+    while(question1!==null){
```

Comment:
無限ループの方法として条件として必ずtueになれば良いので、一番単純なものは以下のように直接trueを指定してしまう方法があります！
```suggestion
    while(true){
```
そうすると、10行目のコードも不要になりますね。